docs/src/content/docs/de/getting-started.mdx

```diff
@@ -5,33 +5,34 @@ description: Lerne wie du deine nächste Dokumentationsseite mit Starlight und A
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
-## Erstelle ein neues Projekt
+Starlight ist ein voll funktionsfähiges Dokumentationsthema, welches auf dem [Astro](https://astro.build) Framework aufbaut.
```

Comment:
Need help with the translation of "documentation theme". Currently chose "Dokumentationsthema" because online translators suggest it but it feels wrong. Not entirely sure what it means in the English version, so I cant find the right German word...